src/js/load-pyodide.js

```diff
@@ -122,8 +126,9 @@ function recursiveDependencies(
   if (sharedLibsOnly) {
     let onlySharedLibs = new Map();
     for (let c of toLoad) {
-      if (c[0] in sharedLibraries) {
-        onlySharedLibs.set(c[0], toLoad.get(c[0]));
+      name = c[0];
+      if (!!Module.packages[name].shared_library) {
```

Comment:
`if(!!x)` is the same as `if(x)`.
```suggestion
      if (Module.packages[name].shared_library) {
```